doc/rbd/iscsi-targets.rst

```diff
@@ -0,0 +1,29 @@
+=============
+iSCSI Targets
+=============
+
+Traditionally, block-level access to a Ceph storage cluster has been
+limited to QEMU and ``librbd``, which is a key enabler for adoption
+within OpenStack environments. Starting with the Ceph Jewel release,
+block-level access is expanding to offer standard iSCSI support allowing
+wider platform usage, and potentially opening new use cases and
+customers.
+
+-  CentOS 7.3 or newer
+
+-  A working Ceph Storage cluster, deployed with ``ceph-ansible`` or using the command-line interface
+
+-  iSCSI gateways nodes, which can either be colocated with OSD nodes or on dedicated nodes
+
+-  Separate network subnets for iSCSI front-end traffic and Ceph back-end traffic
+
+A choice of using Ansible or the command-line interface are the
+available deployment methods for installing and configuring the Ceph
+iSCSI gateway:
+
+-  `Using Ansible`_
+
+-  `Using the Command Line Interface`_
+
+.. _Using Ansible: ../iscsi-target-ansible
```

Comment:
Nit: these aren't included in the toc, so once you click on these two entries, the side menu collapses and you lose your place.